### DISCUSSION TITLE
PAPI: Expansion Error

### DISCUSSION THREAD

**Discussion by LordYarkan** [PlaceholderAPI/PlaceholderAPI#823] [(2) Support]
Hi, I need help.
Lately I'm using a plugin to set a TOP of players with the changeoutput expansion. example:
`%changeoutput_equals_{leaderheads_value_pvplevels_kills_weekly_1}_0_&7-_{leaderheads_name_pvplevels_kills_weekly_1}`

That if they don't have casualties then they won't appear at the top. But the players that have a name with a "_" are cut off and it doesn't give me their real name, for example instead of saying DJ_SAMF it says DJ in the TOP. It does not give me the names correctly that use an underscore, that is, it gives the name incomplete.
that is the problem of ChangeOutput.

**Comment by Andre601** [comment]
This seems to be a prominent issue here... See https://github.com/Kqliber/Expansion-ChangeOutput/issues/2

**Reply by LordYarkan** [reply]
Hello, I see that months have passed with this error and they haven't fixed it then :c
